config/initializers/breakers.rb

```diff
@@ -6,6 +6,7 @@
 require 'central_mail/configuration'
 require 'debt_management_center/debts_configuration'
 require 'decision_review/configuration'
+require 'vye/dgib/configuration'
```

Comment:
require 'vye/dgib/service'

Reply:
Sorry, not clear here, are saying to add an entry for require 'vye/dgib/service' or to replace the existing require 'vye/dgib/configuration' line?

Reply:
Replace configuration with service please

Reply:
`require 'vye/dgib/service'`
`Vye::DGIB::Service.instance.breakers_service`

Reply:
thank you.  changes made.